app/models/customer.rb

```diff
@@ -10,7 +10,12 @@ class Customer < ApplicationRecord
 
 
   def self.top_five_customers_by_merchant
-    joins(:transactions).where('transactions.result = 0').group(:last_name, :first_name).order(count: :desc).limit(5).count
+    joins(:transactions)
```

Comment:
I like how you lined up the "." here to make the code more readable

Reply:
Same! Was wondering if you are able to do that for the admin version as well?